fdbrpc/AsyncFileCached.actor.h

```diff
@@ -278,8 +289,10 @@ class AsyncFileCached final : public IAsyncFile, public ReferenceCounted<AsyncFi
 	Int64MetricHandle countCachePageReadsMerged;
 	Int64MetricHandle countCacheReadBytes;
 
-	AsyncFileCached( Reference<IAsyncFile> uncached, const std::string& filename, int64_t length, Reference<EvictablePageCache> pageCache )
-		: uncached(uncached), filename(filename), length(length), prevLength(length), pageCache(pageCache), currentTruncate(Void()), currentTruncateSize(0) {
+	AsyncFileCached(Reference<IAsyncFile> uncached, const std::string& filename, int64_t length,
+	                Reference<EvictablePageCache> pageCache)
+	  : uncached(uncached), filename(filename), length(length), prevLength(length), pageCache(pageCache),
+	    currentTruncate(Void()), currentTruncateSize(0), rateControl(nullptr) {
```

Comment:
`rateControl` is a `Reference` so there is no need to initialize it.  I had removed this in 6.2 but I guess I missed this detail when I reviewed the merge to 6.3.  Just to prevent future merge conflicts I will fix 6.3 in later PR and it should merge to master cleanly after that.  No need for action here.